src/main/scala/org/virtuslab/internal/YamlError.scala

```diff
@@ -2,7 +2,9 @@ package org.virtuslab.internal
 
 import org.virtuslab.internal.load.reader.token.Token
 
-sealed trait YamlError
+sealed trait YamlError extends Throwable:
```

Comment:
`extends Throwable`?

Reply:
Good point, it should be removed immediately :)